### PR TITLE
Feat: Introduce Divider component #DS-1391

### PR DESCRIPTION
**Pull request by curdaj**

### CODE REVIEW

**Review by pavelklibani — APPROVED** [review]

File: packages/web-react/src/components/Divider/README.md [hunk 1]
```diff
@@ -1,12 +1,9 @@
-# UNSTABLE Divider
+# Divider
 
-> ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
-> Please use it with caution.
-
-The `UNSTABLE_Divider` component is used to separate content in a layout.
+The `Divider` component is used to separate content in a layout.
```
Comment (dlouhak) [inline packages/web-react/src/components/Divider/README.md:3]: What about

```suggestion
The `Divider` component is used as horizontal separator of the content.
```
?

File: packages/web-react/src/components/Divider/stories/Divider.stories.tsx [hunk 2]
```diff
@@ -2,11 +2,11 @@ import { Markdown } from '@storybook/blocks';
 import type { Meta, StoryObj } from '@storybook/react';
 import React from 'react';
 import ReadMe from '../README.md';
-import { UNSTABLE_Divider } from '..';
+import { Divider } from '..';
```
Comment (dlouhak) [inline packages/web-react/src/components/Divider/stories/Divider.stories.tsx:5]:
```sugestion
import { Divider } from '../Divider';
import ReadMe from '../README.md';
```

File: packages/web-react/src/components/index.ts [hunk 3]
```diff
@@ -34,7 +34,7 @@ export * from './TextFieldBase';
 export * from './Toast';
 export * from './Tooltip';
 export * from './UNSTABLE_Avatar';
-export * from './UNSTABLE_Divider';
+export * from './Divider';
```
Comment (dlouhak) [inline packages/web-react/src/components/index.ts:37]: 🔤

File: packages/web-twig/src/Resources/components/Divider/README.md [hunk 4]
```diff
@@ -1,12 +1,9 @@
-# UNSTABLE Divider
+# Divider
 
-> ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
-> Please use it with caution.
-
-The `UNSTABLE_Divider` component is used to separate content in a layout.
+The `Divider` component is used to separate content in a layout.
```
Comment (dlouhak) [inline packages/web-twig/src/Resources/components/Divider/README.md:3]:
```suggestion
The `Divider` component is used as horizontal separator of the content.
```

**Review by dlouhak — APPROVED** [review]

**Review by literat — APPROVED** [review]

**Pull request merged**